lib/diagnostic.js

```diff
@@ -8,22 +8,38 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
+  let result = arguments[0]; // if there are any arguments, 'result' equals the first one. But if not, then it returns undefined
 
+  for (let i = 0; i < arguments.length; i++) {
+    if (result > arguments[i]){
+      result = arguments[i];
+    }
+  }
+  return result;
 };
 
+
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-const sum = function sum() {
-
+const sum = function sum(a,b,c){
+  let result = [];
+  for (let i = 0; i < array.length; i++) {
+    let array[i] = array[()(a + b + c) / 3 )];
+  }
+  return result
 };
```

Comment:
The value we are looking to return here is the sum, not the average. A few things to take note of:
1) line 26: Do you recall the term `arguments`? It's one of the terms that JavaScript provides to us. Because we don't know how many things are going to be passed in, the only we can calculate the sum is to utilize this term.
2) line 27: what is the data type of `result`? you initialized it as an empty array. Should it be an Array?
3) line 28: what is this variable `array`? Where/How did you define it?
4) line 29: this...is a bit of a mess! Recall that the way we access a value inside an array is with `[]`, and the value that goes inside is the index number, which is always an integer, and less than the length of the array itself. What index number is `(a + b + c) / 3`?